integration/image/save_test.go

```diff
@@ -76,7 +77,12 @@ func TestSaveCheckTimes(t *testing.T) {
 	created, err := time.Parse(time.RFC3339, img.Created)
 	assert.NilError(t, err)
 
-	assert.Equal(t, created.Format(time.RFC3339), info.ModTime().Format(time.RFC3339), "expected: %s, actual: %s", created, info.ModTime())
+	if testEnv.UsingSnapshotter() {
+		// containerd archive export sets the mod time to zero.
+		assert.Check(t, is.Equal(info.ModTime(), time.Unix(0, 0)))
+	} else {
+		assert.Check(t, is.Equal(info.ModTime().Format(time.RFC3339), created.Format(time.RFC3339)))
```

Comment:
`info.ModTime().IsZero()` is not the same as `== time.Unix(0, 0)` (`0000-00-00` != `1970-01-01`).


As for using `time.Equal` - in general it should be used instead of plain `==` but I think it's enough for this specific test and it provides a meaningful assert message for us. Worst case we'll get a false negative but that will be easily detectable.

Reply:
> info.ModTime().IsZero() is not the same as == time.Unix(0, 0) (0000-00-00 != 1970-01-01).

Oh! Yes, you're right. Wrote that up without thinking.

>  I think it's enough for this specific test and it provides a meaningful assert message for us.

Yeah, makes sense; that's a good point. Was considering that when writing (whether "correct" vs "usable diff" was worth the trade-off)